src/shared_utils/shared_utils.star

```diff
@@ -46,3 +46,27 @@ def new_port_spec(
         application_protocol=application_protocol,
         wait=wait,
     )
+
+
+def read_file_from_service(plan, service_name, filename):
+    output = plan.exec(
+        service_name=service_name,
+        recipe=ExecRecipe(
+            command=["/bin/sh", "-c", "cat {} | tr -d '\n'".format(filename)]
+        ),
+    )
+    return output["output"]
+
+
+def download_trusted_setup(plan, service_name, output_filepath):
+    plan.exec(
+        service_name=service_name,
+        recipe=ExecRecipe(
+            command=[
+                "wget",
+                "-O",
+                output_filepath,
+                "https://raw.githubusercontent.com/ethereum/c-kzg-4844/main/src/trusted_setup.txt",
```

Comment:
this could be in a constant

Reply:
Tracking in #229 as well

Reply:
I'd not bother to change it to be honest. As we plan to use the upstream (ethereum-genesis-generator) to get all the files that are required to obtain for genesis.